2주차/1920.py

```diff
@@ -0,0 +1,11 @@
+# 1920_수 찾기
+import sys
+
+n=int(sys.stdin.readline())
+nn=set(map(int, sys.stdin.readline().strip().split()))
+
+m=int(sys.stdin.readline())
+mm=list(map(int, sys.stdin.readline().strip().split()))
+
+for num in mm:
+    print(1) if num in nn else print(0)
```

Comment:
```suggestion
    print(int(num in nn))
```
True랑 False를 int로 바꾸면 각각 1, 0이 되니까 이렇게도 쓸 수 있어요
물론 삼항연산자 쓰거나 그냥 if문 쓰는것도 ㄱㅊ